codersdk/experiments.go

```diff
@@ -8,20 +8,12 @@ import (
 
 type Experiment string
 
-const (
-	// ExperimentVSCodeLocal enables a workspace button to launch VSCode
-	// and connect using the local VSCode extension.
-	ExperimentVSCodeLocal Experiment = "vscode_local"
-)
```

Comment:
Could we keep a comment and commented example here for future reference when new ones are added?